addons/website/migrations/9.0.1.0/noupdate_changes.xml

```diff
@@ -9,6 +9,10 @@
     <record id="menu_homepage" model="website.menu">
         <field name="website_id" ref="default_website"/>
     </record>
+    <record id="default_website" model="website">
```

Comment:
This is removed conscientiously, as we don't want this to be changed in your production DB

Reply:
Ok, I will delete now.